### PR TITLE
feat(transition-page): add proceed link

### LINKED ISSUE #551: Introduce a proceed link for users who have verified the go link
**Is your feature request related to a problem? Please describe.**
6s wait is too long for the more tech savvy users who are already familiar and is aware of the need to verify the legitimacy of links. by checking the address bar.

**Describe the solution you'd like**
This solution provide an option for users to proceed without having to wait for the countdown timer to end. 
We've intentionally designed it to be a hyperlink with a string of words so that users will read and proceed, instead of just clicking on a button to proceed.

[Copy yet to be confirmed]
TransitionPage v2
TransitionPage v2

### PR DESCRIPTION
**Pull request by yong-jie**
## Problem

6s wait is too long for the more tech savvy users who are already familiar and is aware of the need to verify the legitimacy of links. by checking the address bar.

Closes #551 

## Solution

This solution provide an option for users to proceed without having to wait for the countdown timer to end.
We've intentionally designed it to be a hyperlink with a string of words so that users will read and proceed, instead of just clicking on a button to proceed.

## Screenshots

**Desktop**:
image

**iPhone 7/8 Plus**
image

**iPhone SE**
image

### CODE REVIEW

File: src/server/views/redirect.ejs [hunk 1]
```diff
@@ -42,3 +47,9 @@ var intervalId = setInterval(function(){
     clearInterval(intervalId)
     proceedToDestination()
 }, 1000)
+
+;(function() {
+    document.getElementById('skip').addEventListener('click', function (e) {
+        proceedToDestination()
+    })
+})()
```
Comment (liangyuanruo) [inline src/server/views/redirect.ejs:55]: never thought i'd have to see an IIFE written

**Review by liangyuanruo — APPROVED** [review]
Code looks OK, to merge pending design's comments, if any.

File: public/assets/transition-page/styles/transition-page.css [hunk 2]
```diff
@@ -81,6 +81,17 @@ body {
   font-size: 13px;
 }
 
+#skip {
+  display: none;
```
Comment (LoneRifle) [inline public/assets/transition-page/styles/transition-page.css:85]:
```suggestion
  opacity: 0;
  display: block;
```

File: src/server/views/redirect.ejs [hunk 3]
```diff
@@ -22,6 +22,11 @@ function proceedToDestination() {
     window.location = document.getElementById('url').getAttribute('data-href')
 }
 
+setTimeout(function(){
+    var skipButton = document.getElementById('skip')
+    skipButton.style.display = 'block'
```
Comment (LoneRifle) [inline src/server/views/redirect.ejs:27]:
```suggestion
    skipButton.style.opacity = '1'
    skipButton.style.transition = 'opacity 0.4s'
```

**Review by LoneRifle — APPROVED** [review]
Consider using a transition to fade the skip link in, so that the visitor notices it more easily

**Pull request merged**